app/views/fsss_view.py

```diff
@@ -24,7 +24,8 @@ def get_emp_info(name, on_false={}, on_missing='Unspecified'):
 
     line_manager = format_line_manager(current_job_role)
 
-    employee_name = map_employee_name(employee_info)
+    
+    employee_name = map_employee_name(employee_info,calledFrom="FSS - employeename")
```

Comment:
Excess blank line, consider running yapf to sort out all the formatting.

The calledFrom here can be removed, it's not used in the function.